examples/cloudplatform/folderiammember.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: cloudplatform.gcp.upbound.io/v1beta1
+kind: FolderIAMMember
+metadata:
+  annotations:
```

My understanding is that we will need a valid folder & member to successfully test this resource. It's okay to manually validate this resource but could you please add the manual-intervention annotation so that uptest skips this resource:
```suggestion
  annotations:
    upjet.upbound.io/manual-intervention: "Depends on a valid folder & member"
```

Or, would it be possible to also add these dependencies to the example manifest and add any necessary references? But if these are organization-level resources, we may not be able to successfully validate these via uptest.